instrumentation/kotlinx-coroutines/javaagent/build.gradle.kts

```diff
@@ -10,25 +10,35 @@ muzzle {
     group.set("org.jetbrains.kotlinx")
     module.set("kotlinx-coroutines-core")
     versions.set("[1.0.0,1.3.8)")
+    extraDependency(project(":instrumentation-annotations"))
+    extraDependency("io.opentelemetry:opentelemetry-api:1.27.0")
```

Comment:
can we also keep the original muzzle definition and use `excludeInstrumentationName`, to make sure the kotlin instrumentation is still applied in the absence of these dependencies on the user classpath?

Reply:
or possibly extract out into separate module?